src/components/info/info.style.js

```diff
@@ -0,0 +1,48 @@
+import styled, { css } from 'styled-components';
+import { breakpoint, mediaQuery } from '../../styles/media-query';
+
+export const InfoWrapper = styled.div`
+  ${mediaQuery(breakpoint.md, css`
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 46.25rem; /* 740px */
+  `)}
+`;
+
+export const Block = styled.article`
+  margin-bottom: 2.5rem; /* 40px */
+
+  ${mediaQuery(breakpoint.md, css`
+    margin-bottom: 5rem; /* 80px */
+  `)}
+`;
+
+export const Heading = styled.h2`
+  margin-bottom: 0.938rem; /* 15px */
+`;
+
+export const List = styled.ul`
+  list-style-type: none;
+  padding: 0;
+`;
+
+export const Item = styled.li`
+  &:before {
+    background-color: ${(props) => props.theme.color.default};
+    border-radius: 50%;
+    content: " ";
+    display: inline-block;
+    vertical-align: middle;
+    height: 0.188rem; /* 3px */
+    width: 0.188rem; /* 3px */
+    margin-right: 0.313rem; /* 5px */
+  }
+`;
+
+export const Link = styled.a`
+  color: ${(props) => props.theme.color.link.default[0]};
+
+  &:hover {
+    color: ${(props) => props.theme.color.link.default[1]};
+  }
```

Comment:
These are the default styles for anchors, right? In that case you can move these styles to the global styles and target the `a` tag

Reply:
The problem with declaring them globally is bad inheritance starts to occur on things like the CTA button which uses an `a` tag (and doesn't want the color to change on hover) and I begin fighting off those styles by having to explicitly declare color states for other components that happen to include `a` tags, if we're assuming a scenario where this project is expanded.

Reply:
That sounds normal to me tbh. A link that looks like a button will have to overwrite some of the default link styles. I'm not sure if there would be so many other scenarios like this. Maybe something like card elements that are wrapped with an anchor and contain text. But also there I'd say it's expected that default styles need to be overwritten (think of the typical `text-decoration: underline` that's often set by default for links)